### PR TITLE
Update WG-prioritization documentation

### PR DESCRIPTION
**Pull request by apiraino**
This is a massive rework of the WG-prioritization documentation. Basically a long standing needed update to the content, also clarified the wording.

Hopefully this will make the document more helpful for contributors.

[Rendered version](https://github.com/rust-lang/rust-forge/blob/292e8adb9f6488ce85f8133243d1f174176b167a/src/compiler/prioritization/procedure.md)

### CODE REVIEW

**Comment by apiraino** [comment]
Hmm unsure who to ping for merging this ... maybe r? @wesleywiser

File: src/compiler/prioritization/procedure.md [hunk 1]
```diff
@@ -1,214 +1,145 @@
...
-We should:
+#### 5. Summarize I-compiler-nominated issues
 
-- Check if these issues were already discussed and in that case remove `I-nominated` label
-- Check if each issue is worth being discussed
-- Add them to the agenda explaining:
+Issues labeled with `I-compiler-nominated` generally are nominated to specifically have the compiler team dedicate them a special slice of the meeting (generally towards the end). After the discussion, add a comment on Github linking the Zulip message where the discussion started (so everyone can read). `T-compiler` sometimes writes a summary of the discussion on the issue itself.
+
+Try to follow the [General issues review process](#general-issues-review-process):
+- Check if an issue is needs a discussion and add the label `I-compiler-nominated`
```
Comment (JohnTitor) [inline src/compiler/prioritization/procedure.md:122]:
```suggestion
- Check if an issue needs a discussion and add the label `I-compiler-nominated`
```

File: src/compiler/prioritization/procedure.md [hunk 2]
```diff
@@ -1,214 +1,145 @@
...
-- [Other team's P-critical](https://github.com/rust-lang/rust/issues?q=is%3Aopen+is%3Aissue+label%3AP-critical+-label%3AT-compiler+-label%3AT-libs)
-- [T-compiler P-high](https://github.com/rust-lang/rust/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3AT-compiler+label%3AP-high+)
-- [T-libs P-high](https://github.com/rust-lang/rust/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3AT-libs+label%3AP-high+)
+The meeting is over! Time to cleanup a little bit.
 
-#### Re-sync and check the agenda right before the meeting
+- Lock the agenda file on HackMD assigning write permissions to `Owners`. Download the markdown file and commit [it to this repository](https://github.com/rust-lang/compiler-team/issues?q=is%3Aall+label%3Amajor-change+label%3Ato-announce).
 
-Re-run the script and re-synchronize contents of the agenda with new information.
+- Remove the `to-announce` label from [ MCPs](https://github.com/rust-lang/compiler-team/issues?q=is%3Aall+label%3Amajor-change+label%3Ato-announce), unless this label was added exactly during the meeting (and therefore will be seen during the following meeting).
```
Comment (JohnTitor) [inline src/compiler/prioritization/procedure.md:139]:
```suggestion
- Remove the `to-announce` label from [MCPs](https://github.com/rust-lang/compiler-team/issues?q=is%3Aall+label%3Amajor-change+label%3Ato-announce), unless this label was added exactly during the meeting (and therefore will be seen during the following meeting).
```

**Review by JohnTitor — APPROVED** [review]
Great, thank you!

Read briefly and overall LGTM with some wording nits.

**Comment by apiraino** [comment]
@JohnTitor can you pls merge if there it we're good? :) thanks

**Comment by JohnTitor** [comment]
Sorry but I don't have the right to do that here (I came here as I found the message on Zulip).

**Comment by apiraino** [comment]
ah sorry my bad. I am always confused about _who_ has _which_ rights :)

perhaps @rylev  can help for merging?

**Review by rylev — APPROVED** [review]

**Pull request merged**